lib/tasks/tol_tubes_report_view.rake

```diff
@@ -70,4 +70,52 @@ namespace :tol_tubes_report_view do
       puts '-> TOL tubes report view successfully created' unless Rails.env.test?
     end
   end
+
+  namespace :v2 do
```

Comment:
I think this also needs to be called in a migration.  Similar to https://github.com/sanger/traction-service/blob/develop/db/migrate/20240219180656_create_cost_code_and_study_report_view.rb but with `up` calling `v2:create` and down calling `v1:create`

Reply:
I normally run these kind of rake tasks directly on the servers. Happy to add a migration for it if you think that is the correct practice. It is tied to the irreversible migration where pacbio_library loses its pool_id association, so maybe I should call it as part of that?

Reply:
Generally I think it's right to run these Rake tasks directly on the servers, but this one is modifying the database.  Unfortunately Rails doesn't support view management natively, so we did it this way instead.  I think you should add it to your existing migration file since it's tied to those columns being modified.